tests/apollo/test_skvbc_long_running.py

```diff
@@ -60,4 +61,8 @@ class SkvbcLongRunningTest(unittest.TestCase):
                 await trio.sleep(seconds=10)
                 await SkvbcFastPathTest().test_fast_path_read_your_write \
                     (bft_network=bft_network, already_in_trio=True, disable_linearizability_checks=True)
-                await trio.sleep(seconds=10)
\ No newline at end of file
+                await trio.sleep(seconds=10)
+                if i % 250 == 0:
```

Comment:
250 looks like what we would call a "magic number". Where does it come from? Does it depend on the overall long run test duration?